test/functional/workflows_controller_test.rb

```diff
@@ -416,7 +416,7 @@ def setup
       post :create_ro_crate, params: {
           ro_crate: {
               workflow: { data: fixture_file_upload('files/checksums.txt') },
-              diagram: { data: fixture_file_upload('files/file_picture.png') },
+              #             diagram: { data: fixture_file_upload('files/file_picture.png') },
```

Comment:
Is there a reason for these being commented out?

Reply:
That is what was causing the diagram route complaints that I never sorted out

Reply:
Uncommented. Now checking if it builds

Reply:
It didn't and I cannot sensibly debug.